guides/predictive-analytics-with-deepnote.md

```diff
@@ -0,0 +1,135 @@
+2. Connect to your Hydra database and import the CSV file to the database using the steps for importing a CSV file outlined in the [Documentation](https://docs.hydra.so/centralize-data/load/from-local-csv-file)
+
+### Creating the Connection
+
+Now we can connect Deepnote to our sample data in Hydra.
+
+1. Start by [signing up](https://deepnote.com/sign-up) for Deepnote if you haven't done so already.
+2. Within your Deepnote account, create a workspace name. In the case of this tutorial, the workspace name will be "Hydra-Team".
+3. Next, choose 'PostgreSQL' as the data source as shown in the image below.
+
+<figure><img src="../.gitbook/assets/.predictive-analytics/datasource.png" alt=""><figcaption></figcaption></figure>
+
+4. Create a new project by clicking on '+' under 'Projects' within the left panel of the Deepnote console.
+
+<figure><img src="../.gitbook/assets/.predictive-analytics/create project.png" alt=""><figcaption></figcaption></figure>
+
+5. Within your new project, navigate to 'Integrations' on the right panel and choose 'PostgreSQL'.
+
+<figure><img src="../.gitbook/assets/.predictive-analytics/Integrating postgres.png" alt=""><figcaption></figcaption></figure>
+
+6. At the 'Connect to PostgresSQL' popup, add your Hydra database credentials including Hostname, User, Password, and Database.
+
+<figure><img src="../.gitbook/assets/.predictive-analytics/connecting postgres.png" alt=""><figcaption></figcaption></figure>
+
+7. After establishing the connection, you can view the imported salary table within Deepnote.
+
+Using the SELECT statement, you are able to run SQL queries on the salary table which enables you to retrieve and analyze specific data from the table.
+
+```sql
+SELECT * FROM public.salary LIMIT 100
+```
+
+<figure><img src="../.gitbook/assets/.predictive-analytics/Hydra dashboard.png" alt=""><figcaption></figcaption></figure>
+
+8. To access the table using Python scripts because predictions can not be done using SQL, an engine has to be created with the data warehouse credentials
+
+```python
+import pandas as pd
+from sqlalchemy import create_engine
+#user = username
+#pass = password
+#server = hostname
+#database = database-name
+engine = create_engine('postgresql://user:pass@server:5432/database')
+#table_name = Salary
+#database = public
+data = pd.read_sql_table('table_name',con= engine, schema= 'database')
+```
+
+**Congratulations!** You've successfully connected Hydra with Deepnote! What next?
+
+## Making the predictions
+
+Now that we have connected Deepnote to Hydra, we can use Python and SQL scripts to access the data. We will use the Linear Regression model to create predictions leveraging the salary data.
+
+The goal of linear regression model is to find a mathematical relationship or equation that best fits the data and helps us predict salaries based on the other factors.
```

Comment:
suggest combining these into a single paragraph.

Reply:
You suggested removing the second paragraph in the earlier comment, do I need to go ahead to combine it or remove it?